test/integration2/schema/foreign-keys.spec.js

```diff
@@ -0,0 +1,78 @@
+const { expect } = require('chai');
+const { getAllDbs, getKnexForDb } = require('../util/knex-instance-provider');
+
+describe.skip('Schema', () => {
```

Comment:
This is an example test that is also something that @rijkvanzanten can use for their own PR, it's also a PoC for the new mechanism.